StripePaymentSheet/StripePaymentSheet/Source/Helpers/StaticEphemeralKeyProvider.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  StaticEphemeralKeyProvider.swift
+//  StripePaymentSheet
+//
+//  Created by David Estes on 3/13/23.
+//
+
+import Foundation
+@_spi(PrivateBetaSavedPaymentMethodsSheet) import StripePaymentsUI
+
+@_spi(PrivateBetaSavedPaymentMethodsSheet) public extension _stpspmsbeta_STPCustomerContext {
```

Comment:
Hmm If we really don't want customers to use this --  should this be part of our playground package instead?

Reply:
I think the convenience init is useful to users, it makes it easy to create an STPCustomerContext with the same values you'd use for a PaymentSheet Customer Settings struct

Reply:
I see- then we should update the comment to:
    /// This is not recommended, use initWithKeyProvider and an STPCustomerEphemeralKeyProvider instead.
    /// The key will expire after 60 minutes.

Reply:
Makes sense!